### ISSUE TITLE
Fix the formatting 

### ISSUE THREAD

**Issue by ajeetraina** [collabnix/docker-community-extensions#152]
@sivolko Can you review the table and fix the issue with the layout?

**Comment by sivolko** [comment]
> @sivolko Can you review the table and fix the issue with the layout?

Yes, I can see Storage , Logging section is broken and security part is getting carousel . I'm fixing it

**Comment by sivolko** [comment]
resolved

**Comment by ajeetraina** [comment]
I still issue with dockerextensions.collabnix.com page esp for 
<img width="839" alt="Screenshot 2023-12-02 at 11 44 41 AM" src="https://github.com/collabnix/docker-community-extensions/assets/313480/8bdb3085-bc41-4c78-9913-9f83fc8d297f">

**Comment by sivolko** [comment]
Resolved Now
